examples/examples_uq/plot_nas_deep_ensemble_uq_regression_pytorch.py

```diff
@@ -752,10 +760,9 @@ def run_neural_architecture_search(problem, max_evals):
 
     evaluator = Evaluator.create(
         run,
-        method="ray",  
+        method="thread",  
```

Comment:
an `if` statement can be used here to check:
- if device = cuda then method = thread, num_workers = 1 (unless we find out that multiple trainings can be run in parallel on the same gpu!)
- if device = cpu then checp `multiprocessing.cpu_count()` and set `method="ray"`, `num_cpus=...` based ont the `cpu_count()`, `num_cpus_per_task=1`.